Cargo.toml

```diff
@@ -27,36 +27,36 @@ cfg-if = "= 0.1.2"
 
 [dependencies.algebra]
 git = "https://github.com/ZencashOfficial/ginger-lib.git"
-rev = "69f3c0b2f80a90379587fa5a5cb0e9be130cba6e"
+rev = "9ef4769b976f7b6df15cd6b4c59bbd3176fa0c63"
```

Comment:
Change to last commit on GingerLib's development branch, when corresponding PR in GingerLib will be approved and merged in development branch.